apps/demo/emails/airbnb-review.tsx

```diff
@@ -0,0 +1,159 @@
+import { Button } from '@react-email/button';
+import { Container } from '@react-email/container';
+import { Head } from '@react-email/head';
+import { Hr } from '@react-email/hr';
+import { Html } from '@react-email/html';
+import { Img } from '@react-email/img';
+import { Link } from '@react-email/link';
+import { Preview } from '@react-email/preview';
+import { Section } from '@react-email/section';
+import { Text } from '@react-email/text';
+import * as React from 'react';
+
+export default function Email() {
+  return (
+    <Html>
+      <Head />
+      <Preview>Read Alex's review</Preview>
+      <Section style={main}>
+        <Container style={container}>
+          <Img
+            src="https://assets.react.email/demo/airbnb-logo.png"
+            height="80"
+            alt="Airbnb"
+          />
+          <Section>
+            <Img
+              src="https://assets.react.email/demo/airbnb-review-user.jpg"
+              width="96"
+              height="96"
+              alt=""
+              style={userImage}
+            />
+          </Section>
+          <Text style={heading}>Here's what Alex wrote</Text>
+          <Text style={review}>
+            “Zeno was a great guest! Easy communication, the apartment was left
+            in great condition, very polite, and respectful of all house rules.
+            He’s welcome back anytime and would easily recommend him to any
+            host!”
+          </Text>
+          <br />
+          <Text style={paragraph}>
+            Now that the review period is over, we’ve posted Alex’s review to
+            your Airbnb profile.
+          </Text>
+          <Text style={paragraph}>
+            While it’s too late to write a review of your own, you can send your
+            feedback to Alex using your Airbnb message thread.
+          </Text>
+          <div style={{ padding: '16px 0 24px 0' }}>
```

Comment:
Can we not depend on a `div` here?

Reply:
Could be `<Section />` here? or something else?

Reply:
Yeah, it could!